From e5660cce054a8c9ce4fb50ae62614221b56ec5b2 Mon Sep 17 00:00:00 2001
From: Eddielu <Eddie.Lu@quantatw.com>
Date: Wed, 10 Jun 2020 16:57:03 +0800
Subject: [PATCH] Add the paths used with Bitbake in GBS

1.Modify FIU0_DRD_CFG_Set register
2.Modify FIU_Clk_Divider

---
 ImageGeneration/references/BootBlockAndHeader_EB.xml | 4 ++--
 ImageGeneration/references/UbootHeader_EB.xml        | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/ImageGeneration/references/BootBlockAndHeader_EB.xml b/ImageGeneration/references/BootBlockAndHeader_EB.xml
index 775534f..f769ee1 100644
--- a/ImageGeneration/references/BootBlockAndHeader_EB.xml
+++ b/ImageGeneration/references/BootBlockAndHeader_EB.xml
@@ -63,7 +63,7 @@
 			<offset>0x108</offset>       
 			<size>0x4</size> 
 		</config>
-		<content format='32bit'>0x030011BB</content>               <!-- content the user should fill -->
+		<content format='32bit'>0x0300100B</content>               <!-- content the user should fill -->
 	</BinField>
 	
 	<BinField>
@@ -73,7 +73,7 @@
 			<offset>0x10C</offset>       
 			<size>0x1</size> 
 		</config>
-		<content format='bytes'>4</content>               <!-- content the user should fill -->
+		<content format='bytes'>0x0A</content>               <!-- content the user should fill -->
 	</BinField>
 	
 	<BinField>
diff --git a/ImageGeneration/references/UbootHeader_EB.xml b/ImageGeneration/references/UbootHeader_EB.xml
index 1e72e22..4434504 100644
--- a/ImageGeneration/references/UbootHeader_EB.xml
+++ b/ImageGeneration/references/UbootHeader_EB.xml
@@ -63,7 +63,7 @@
 			<offset>0x108</offset>        <!-- offset in the header -->
 			<size>0x4</size>              <!-- size in the header -->
 		</config>
-		<content format='32bit'>0x030111BC</content>               <!-- content the user should fill 0x030032EB -->
+		<content format='32bit'>0x0300100B</content>               <!-- content the user should fill 0x030032EB -->
 	</BinField>
 	
 	<BinField>
-- 
2.7.4

